docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,590 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "exp1.analysis.set_options(p0=None, plot=True)\n",
+    "expdata1 = exp1.run(backend=backend, shots=2000)\n",
+    "expdata1.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "display(expdata1.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DbAnalysisResultV1\n",
+      "- name: @Parameters_T2HahnAnalysis\n",
+      "- value: [-2.31726995e+03  2.31808006e+03 -5.66599325e-01] ± [nan nan nan]\n",
+      "- χ²: 45.44971077076316\n",
+      "- quality: bad\n",
+      "- extra: <4 items>\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n",
+      "DbAnalysisResultV1\n",
+      "- name: T2\n",
+      "- value: -0.5665993249904158 ± nan s\n",
+      "- χ²: 45.44971077076316\n",
+      "- quality: bad\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print results\n",
+    "for result in expdata1.analysis_results():\n",
+    "    print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Providing initial user estimates\n",
+    "The user can provide initial estimates for the parameters to help the analysis process. Because the curve is expected to decay toward $0.5$, the natural choice for parameter $B$ is $0.5$. Since if there was no delay we would expect that the probability to measure `1` is $100\\%$, so we will guess that A is $0.5$. In this experiment, `t2hahn` is the parameter of interest. Good estimate for it is the value computed in previous experiments on this qubit or a similar value computed for other qubits."
```

Comment:
changed as following:
`The user can provide initial estimates for the parameters to help the analysis process. Because the curve is expected to decay toward $0.5$, the natural choice for parameter $B$ is $0.5$. When there is no T2 error, we would expect that the probability to measure `1` is $100\\%$, we will guess A to be $0.5$. In this experiment, `t2hahn` is the parameter of interest. Good estimate for it is the value computed in previous experiments on this qubit or a similar value computed for other qubits."`